openkfw/TruBudget
# storage-service:Add missing env vars

### Checklist




### Description


### How to test

1. Start trubudget with operation script
2. Verify that file storage functionality is working
